StripeFinancialConnections/StripeFinancialConnections/Source/Web/FinancialConnectionsWebFlowViewController.swift

```diff
@@ -293,9 +291,24 @@ extension FinancialConnectionsWebFlowViewController {
 
         notifyDelegate(result: .failed(error: error))
     }
+}
 
-    private static func extractValue(from url: URL, key: String) -> String? {
-        guard let components = URLComponents(url: url, resolvingAgainstBaseURL: false) else {
+private extension URL {
+    
+    func extractLinkBankPaymentMethod() -> LinkBankPaymentMethod? {
```

Comment:
Minor: A code comment here explaining that this object comes base64 encoded would be nice, since it's unlike the other return URL parameters!

Reply:
Will do!